docs/index.rst

```diff
@@ -40,6 +39,16 @@ encoding
   Encoding to be used to read an OpenAPI spec. If not passed, Sphinx's
   source encoding will be used.
 
+paths
+  A comma separated list of paths to filter the included openapi spec by. For example:
+
+  .. code:: restructuredtext
+
+      .. openapi:: specs/openapi.yml
+          :paths: /persons,/evidence
```

Comment:
I used `paths` because that's the name used by the OpenAPI specification, and I wanted to be consistent with that. But if `endpoints` is more consistent with your documentation that seems ok too.

:+1: on the 3-space indentation.

Definitely will switch to split by line, that reads better and won't break if for some crazy reason someone used a comma in their URL.